Sofa/GUI/Component/src/sofa/gui/component/performer/TopologicalChangeManager.h

```diff
@@ -136,6 +137,7 @@ class SOFA_GUI_COMPONENT_API TopologicalChangeManager
 
     Index removeItemsFromTriangleModel(sofa::component::collision::geometry::TriangleCollisionModel<sofa::defaulttype::Vec3Types>* model, const type::vector<Index>& indices) const;
     Index removeItemsFromPointModel(sofa::component::collision::geometry::PointCollisionModel<sofa::defaulttype::Vec3Types>* model, const type::vector<Index>& indices) const;
+    Index removeItemsFromLineModel(sofa::component::collision::geometry::LineCollisionModel<sofa::defaulttype::Vec3Types>* model, const type::vector<Index>& indices) const;
```

Comment:
I know this is not done for the rest of the header, but could you add a small doxygen doc like:
```
/** \brief Method to remove topological elements from a Topology linked to a Line collision model. Only Edge Topology  is supported.
*  \param indices : list of element indices to remove (unique check is done internally)
*/ @
```